Source/WarpX.H

```diff
@@ -861,28 +1059,33 @@ public:
      * \param[in] fy_parser parser function to initialize y-field
      * \param[in] fz_parser parser function to initialize z-field
      * \param[in] lev level of the Multifabs that is initialized
-     * \param[in] patch_type PatchType on which the field is initialized (fine or coarse)
-     * \param[in] eb_update_field flag indicating which gridpoints should be modified by this functions
-     * \param[in] use_eb_flags (default:true) flag indicating if eb points should be excluded or not
+     * \param[in] patch_type PatchType on which the field is initialized (fine
```

Comment:
While I like the idea of stetting a limit for the number of columns used for docstrings, I think that we may need an extra indentation here for the description of the input parameters.

Reply:
Like:
```
\param[in] patch_type PatchType on which the field is initialized (fine
                    or coarse)
```

Reply:
I am not sure if it is possible to customise this in a way that is systematic, beyond wrapping the relevant lines in a "don't format" wrap.